protos/orchestrator_service.proto

```diff
@@ -306,13 +306,18 @@ message OrchestratorRequest {
     repeated HistoryEvent pastEvents = 3;
     repeated HistoryEvent newEvents = 4;
     OrchestratorEntityParameters entityParameters = 5;
+    bool requiresHistoryStreaming = 6;
```

Comment:
why not a required cabailities repeated enum field that repurposes the worker capabilities enum?

Reply:
The more I think about it - I think this is fine here. Even required capabilities doesn't semantically mean that the worker would then exercise that capability - just that it would support this.

Since this is telling the worker that it must fetch the orchestration history via streaming - I think having this boolean property here makes sense.

Reply:
This is not a capabilities flag. Rather, this is a signal to the worker that we've intentionally omitted the history (`pastEvents`) from this particular work item and it's up to you (the worker) to go get the history yourself (e.g., via the new `StreamInstanceHistory` API). This flag will only ever be set to `true` when the worker connects and includes the `WORKER_CAPABILITY_HISTORY_STREAMING ` flag.